examples/aflow/experiments/optimize_mbpp.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# @Date    : 8/23/2024 20:00 PM
+# @Author  : didi
+# @Desc    : Entrance of AFlow.
+
+from metagpt.configs.models_config import ModelsConfig
+from metagpt.ext.aflow.scripts.optimizer import DatasetType, Optimizer, QuestionType
+
+# Crucial Parameters
```

Comment:
Use distinct configuration files and argparse to handle benchmark-specific settings, allowing flexibility and avoiding multiple independent run entry points. This will streamline experiment setup and improve code maintainability.